src/Servers/Kestrel/test/Interop.FunctionalTests/Http3/Http3TlsTests.cs

```diff
@@ -61,6 +63,179 @@ public async Task ServerCertificateSelector_Invoked()
+        [InlineData(ClientCertificateMode.DelayCertificate)]
+        [MsQuicSupported]
+        public async Task ClientCertificate_NoOrDelayed_Available_Ingored(ClientCertificateMode mode)
+        {
+            var builder = CreateHostBuilder(async context =>
+            {
+                var hasCert = context.Connection.ClientCertificate != null;
+                await context.Response.WriteAsync(hasCert.ToString());
+            }, configureKestrel: kestrelOptions =>
+            {
+                kestrelOptions.ListenAnyIP(0, listenOptions =>
+                {
+                    listenOptions.Protocols = HttpProtocols.Http3;
+                    listenOptions.UseHttps(httpsOptions =>
+                    {
+                        httpsOptions.ServerCertificate = TestResources.GetTestCertificate();
+                        httpsOptions.ClientCertificateMode = mode;
+                        httpsOptions.AllowAnyClientCertificate();
+                    });
+                });
+            });
+
+            using var host = builder.Build();
+            using var client = Http3Helpers.CreateClient(includeClientCert: true);
+
+            await host.StartAsync().DefaultTimeout();
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"https://127.0.0.1:{host.GetPort()}/");
+            request.Version = HttpVersion.Version30;
+            request.VersionPolicy = HttpVersionPolicy.RequestVersionExact;
+
+            var response = await client.SendAsync(request, CancellationToken.None).DefaultTimeout();
+            response.EnsureSuccessStatusCode();
+            var result = await response.Content.ReadAsStringAsync();
+            Assert.Equal(HttpVersion.Version30, response.Version);
+            Assert.Equal("False", result);
+
+            await host.StopAsync().DefaultTimeout();
+        }
+
+
+        [ConditionalTheory]
+        [InlineData(ClientCertificateMode.RequireCertificate)]
+        [InlineData(ClientCertificateMode.AllowCertificate)]
+        [MsQuicSupported]
+        public async Task ClientCertificate_AllowOrRequire_Available_Invalid_Refused(ClientCertificateMode mode)
+        {
+            var builder = CreateHostBuilder(async context =>
+            {
+                var hasCert = context.Connection.ClientCertificate != null;
+                await context.Response.WriteAsync(hasCert.ToString());
+            }, configureKestrel: kestrelOptions =>
+            {
+                kestrelOptions.ListenAnyIP(0, listenOptions =>
+                {
+                    listenOptions.Protocols = HttpProtocols.Http3;
+                    listenOptions.UseHttps(httpsOptions =>
+                    {
+                        httpsOptions.ServerCertificate = TestResources.GetTestCertificate();
+                        httpsOptions.ClientCertificateMode = mode;
+                        // httpsOptions.AllowAnyClientCertificate(); // The self-signed cert is invalid. Let it fail the default checks.
```

Comment:
Meant to still be here?

Reply:
This is the only line that's different from the prior test so I wanted to call it out rather than delete it.